aip/general/0008.md

```diff
@@ -191,12 +209,32 @@ subdirectory); this ensures that the link works both on the AIP site, when
 viewing the Markdown file on GitHub, using the local development server, or a
 branch.
 
+## Rationale
+
+### Designing for a broad set of clients
+
+API guidance, similar to any software, is most beneficial when there is a clear
+purpose and target beneficiary.
+
+The beneficiaries of improved API design are users. These users interact with
+APIs via a variety of methods, depending on their use case:
+
+- Command line interfaces for exploration and simple automation.
+- [IaC][] clients for orchestration and automation of multiple resources.
+- Visual User Interfaces for visualization and one-off manual actions.
+- SDKs for data-plane operations.
+
+API guidance must in turn consider each of these different clients.
```

Comment:
I'd say a "broad range of types of clients", to tie it back to the subsection header.

Reply:
I went with "these clients" to refer to the above list. WDYT?

Reply:
actually I did some wordsmithing and moved the list to joint it with the more comprehensive list in the guidance. PTAL.